### DISCUSSION TITLE
Error message : Ziggy is not defined

### DISCUSSION THREAD

**Discussion by d-stephane** [tighten/ziggy#753] [Q&A]
Hello

I have the following error when loading my page:

``Uncaught ReferenceError: Ziggy is not defined``

In my <head> I added the @routes directive before calling JS files

I use the following versions:
- Laravel 11.7 (php 8.3)
- Ziggy 2.1

Any idea?

**Comment by bakerkretzmar** [comment]
Gonna need more information to help debug this. Can you share code samples? When/where are you seeing that error?

**Comment by d-stephane** [comment]
I just reviewed all my template and I just saw that I made a stupid mistake...
So everything works out...
So sorry.
I have to go to sleep:)
